nemo/backends/pytorch/actions.py

```diff
@@ -919,10 +919,10 @@ def __module_export(
         dynamic_axes = defaultdict(list)
 
         def __extract_dynamic_axes(port_name: str, ntype: NeuralType, dynamic_axes: defaultdict):
```

Comment:
`dynamic` means the same as `is_list` in Axis definition? if so, can we unify those names?

Reply:
no. this is "dynamic axis" concept from ONNX - typical examples - batch, time.